packages/@uppy/robodog/src/upload.js

```diff
@@ -14,12 +14,17 @@ function upload (files, opts = {}) {
   addTransloaditPlugin(uppy, opts)
 
   files.forEach((file) => {
-    uppy.addFile({
-      data: file,
-      type: file.type,
-      name: file.name,
-      meta: file.meta || {}
-    })
```

Comment:
Not sure about this one, should it be wrapped in try/catch, @goto-bus-stop?

Reply:
This loop seems ok?

Reply:
Oh right. I think we should keep throwing here, even for restriction errors, you want to know that kind of thing in the programmatic API

Reply:
Right, ok! Guessing you meant leaving it as it was, so not logging here at all.